packages/ctl/ctl.js

```diff
@@ -300,6 +395,117 @@ instanceConfigCommands.add(new libCommand.Command({
 		await libLink.messages.setInstanceConfigProp.send(control, request);
 	},
 }));
+
+instanceConfigCommands.add(new libCommand.Command({
+	definition: ["edit <instance> [editor]", "Edit instance configuration", (yargs) => {
+		yargs.positional("instance", { describe: "Instance to set config on", type: "string" });
+		yargs.positional("editor", { describe: "Editor to use",
+			type: "string",
+			default: "" });
+	}],
+	handler: async function(args, control) {
+		let instanceId = await libCommand.resolveInstance(control, args.instance);
+		let response = await libLink.messages.getInstanceConfig.send(control, { instance_id: instanceId });
+		let tmpFile = await libFileOps.getTempFile("ctl-", "-tmp", os.tmpdir());
+		let editor = 0;
+		if (args.editor) {
```

Comment:
Editor selection is straight up duplicated from master, maybe make it a function?

Reply:
I was under the impression the editor launching needed to be in the same context as the later .on method, but  I will look to see if it can be functionised at least somewhat.